src/app/views/dashboard/components/ActivityGrid.jsx

```diff
@@ -69,6 +69,9 @@ const activityTypes = {
 const printDetails = (activity) => {
   if (activityTypes[activity.slug]) {
     if (activityTypes[activity.slug].render) {
+      if (activity.day === null) {
+        activity.day = '"Not Especified"';
```

Comment:
#### **Idea:** in case activity has more key with null result
```js
Object.keys(activity).map(el => {
   if(activity[el] === null){
      activity[el] = '"Not Especified"'
   }
   return activity[el]
   }
)
```

Reply:
Ariel como puedo aplicar esta idea en el codigo?